src/pilot/templates/nic-configs/4_port/controller.yaml

```diff
@@ -167,56 +200,67 @@ resources:
                   type: linux_bond
                   name: bond0
                   bonding_options: {get_param: ControllerBondInterfaceOptions}
+                  mtu: {get_param: DefaultBondMTU}
                   members:
                     -
                       type: interface
                       name: {get_param: ControllerBond0Interface1}
+                      mtu: {get_param: DefaultBondMTU}
                       primary: true
                     -
                       type: interface
                       name: {get_param: ControllerBond0Interface2}
+                      mtu: {get_param: DefaultBondMTU}
                 -
                   type: vlan
                   device: bond0
-                  vlan_id: {get_param: TenantNetworkVlanID}
+                  vlan_id: {get_param: TenantNetworkVlanID
```

Comment:
It looks like a closing "}"was accidentally deleted at the end of the above line.